art-Creating_a_Service_for_VM_Provisioning/topics/Examining_the_Provisioning_Dialog.adoc

```diff
@@ -0,0 +1,11 @@
+[[provisioning-dialog]]
+== Examining the Provisioning Dialog
+
+Understanding provisioning dialogs is important to work with services in {product-title_short}. Every provider in {product-title_short} comes with a sample provisioning dialog covering the options specific to that provider. The process begins with the provisioning dialog, which presents the basic configuration options for a virtual machine or instance. The options presented are dependent on the provider you are using. For instance, a cloud provider might have flavors of instances, whereas an infrastructure provider might allow you to set the memory size or number of CPUs on a virtual machine.
```

Comment:
Suggested revision of the first sentence here: "Understanding provisioning dialogs is important when working with.."